iib/workers/tasks/build.py

```diff
@@ -41,7 +42,6 @@
 
 __all__ = ['handle_add_request', 'handle_rm_request']
 
-
```

Comment:
Accidental?

Reply:
Maybe OCD? :stuck_out_tongue: 

But for me it's ok to remove this line, black doesn't complain as well.